person_capture/gui_app.py

```
@@ -2034,21 +2035,33 @@ def _ratio_list_for_profile(profile: str) -> list[str]:
+            # Keep 3:2 available only for body/context profiles after the scorer
+            # confirms the current frame is actually suited to it.
             allow_landscape = profile == "body"
+            available = validated_user_ratios if validated_user_ratios else preferred
             out: list[str] = []
-            for rs in preferred:
+
+            def _add_ratio(rs: str) -> None:
                 try:
                     rw, rh = parse_ratio(rs)
                     aspect = float(rw) / max(1e-6, float(rh))
                 except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)